lib/ember-test-helpers/test-module.js

```diff
@@ -230,6 +230,13 @@ export default Klass.extend({
     this.container = items.container;
     this.registry = items.registry;
 
+
+    var viewRegistry = this.container.lookup('-view-registry:main');
+    if (viewRegistry) {
+      var View = Ember.__loader.require('ember-views/views/view').default;
+      View.views = viewRegistry;
```

Comment:
Ah, I think I see the problem. ember-test-helpers is doing a `Component.create`, and although it passes in a container, [that is insufficient.](https://github.com/emberjs/ember.js/blob/df9fba3f19874d9f64d7d176be4803bbf8223278/packages/ember-views/lib/views/view.js#L1323)

To play nicely with dependency injection, ember-test-helpers should be doing lookupFactory instead. Is there a generic component factory available? Something equivalent to the old `view:default`?